app/controllers/scaptimony_policies_controller.rb

```diff
@@ -83,6 +83,17 @@ def update_multiple_hosts
     end
   end
 
+  def disassociate_multiple_hosts; end
+
+  def remove_policy_from_multiple_hosts
+    if (id = params['policy']['id'])
```

Comment:
two notes here
1. `params.fetch(:policy, {})[:id]` would be a slightly safer
2. it seems we don't handle else case anyway